ws/public/main.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+(function () {
+    const canvas = document.getElementsByClassName('whiteboard')[0]
```

Comment:
nit: I would find it more conceptually correct that we reference this element by id, as there is only one ever supposed to exist in the app

Reply:
I used id now instead. for me the id seems very "powerful" to use